packages/polaris-viz/src/components/FunnelChart/Chart.tsx

```diff
@@ -57,6 +57,10 @@ export function Chart({
   const xValues = dataSeries.map(({key}) => key) as string[];
   const yValues = dataSeries.map(({value}) => value) as [number, number];
 
+  if (!isFinite(Math.max(...yValues))) {
```

Comment:
We're doing another `Math.max(...yValues)])` further down for creating the yScale. Can we create a variable above for the check and the yScale so we're not doing this twice?